tests/unit/components/cylc/tree/tree.spec.js

```diff
@@ -186,6 +186,43 @@ describe('CylcTree', () => {
       expect(cylcTree.root.children[0].id).to.equal(`${WORKFLOW_ID}|2`)
       expect(cylcTree.root.children[1].id).to.equal(`${WORKFLOW_ID}|1`)
     })
+    it('Should add cycle points sorted DESC by default', () => {
```

Comment:
These two tests assume we have the DESC order by default for cycle points. Should we change it later, we just update the constant in `CylcTree`, and these two tests, plus the remaining tests that are expecting elements in order.